learntools/embeddings/nbs/4-tsne.ipynb

```diff
@@ -194,6 +196,13 @@
     "For example, all the Harry Potter movies should be close together, right?"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### DB: It's worth thinking about whether there's a simpler way to do this visualization (e.g. in Seaborn). The goal of these explanations is for users to be able to do similar work in their own projects, and very few users know enough matplotlib to do this level of detail (and we don't teach enough for them to learn this)."
```

Comment:
Hm, in terms of matplotlib APIs used, I feel like this is pretty basic. We're mostly just calling `axis.scatter()` with some keyword args. The adding of text labels (and adjusting them with `adjust_text`) in `plot_with_annotations` is maybe a little advanced, but I think it's still pretty grokkable (and it's only used for one plot).

It's possible we could implement these with a bit less code using seaborn, but I'm not convinced it would be a big savings (in terms of lines of code, or complexity). I think most of the complexity here is intrinsic rather than incidental.

Reply:
Are there particular parts that you think are going to be confusing? It's been a while since I looked at the lessons in the visualization track, so I don't have a great recollection of what's covered there.